compiler/rustc_codegen_ssa/src/target_features.rs

```diff
@@ -105,6 +105,8 @@ const AARCH64_ALLOWED_FEATURES: &[(&str, Option<Symbol>)] = &[
     ("paca", None),
     // FEAT_PAUTH (generic authentication)
     ("pacg", None),
+    // FEAT_PAUTH
+    ("pauth", None),
```

Comment:
We intentionally split `pauth` into `paca` and `pacg` (https://github.com/rust-lang/rust/pull/93782). why not use them?

Also, this adds stable target feature, but I'm not sure if adding a new target feature without a period of unstable is acceptable.